ice-hockey-manager-service/src/main/java/cz/muni/fi/pa165/service/facade/TeamFacade.java

```diff
@@ -0,0 +1,20 @@
+package cz.muni.fi.pa165.service.facade;
+
+import cz.muni.fi.pa165.dto.TeamCreateDTO;
+import cz.muni.fi.pa165.dto.TeamDTO;
+import cz.muni.fi.pa165.enums.CompetitionCountry;
+
+import java.util.List;
+
+/**
+ * @author Lukáš Kotol
+ */
+public interface TeamFacade {
```

Comment:
Javadoc in facade interface is important. Moreover interface's methods have public access by default.